lms/djangoapps/instructor_task/api.py

```diff
@@ -280,6 +280,13 @@ def submit_bulk_course_email(request, course_key, email_id):
     # the InstructorTask status.
     email_obj = CourseEmail.objects.get(id=email_id)
     targets = [target.target_type for target in email_obj.targets.all()]
+    #task_input has a limit to the size it can store, so cohorts are combined and counted
+    num_cohorts = targets.count('cohort')
+    targets = set(targets)  # remove duplicate 'cohort' entries
```

Comment:
FYI: [`collections.Counter`](https://docs.python.org/2/library/collections.html#collections.Counter) can make this a little smoother:
    
    >>> from collections import Counter
    >>> target_counts = Counter(['cohort', 'cohosh', 'cohousing', 'cohort', 'cohort', 'cohoes'])
    >>> target_counts
    Counter({'cohort': 3, 'cohosh': 1, 'cohousing': 1, 'cohoes': 1})
    >>> target_counts['cohort']
    3
    >>> target_counts['cocoapuffs']  # works even the object isn't included
    0
    >>> target_counts.keys()
    ['cohort', 'cohosh', 'cohousing', 'cohoes']